vms/platformvm/service.go

```diff
@@ -823,6 +823,12 @@ func (s *Service) GetCurrentValidators(_ *http.Request, args *GetCurrentValidato
 		}
 		potentialReward := json.Uint64(currentStaker.PotentialReward)
 
+		potentialDelegateeReward, err := s.vm.state.GetDelegateeReward(currentStaker.SubnetID, currentStaker.NodeID)
+		if err != nil {
+			potentialDelegateeReward = 0
+		}
```

Comment:
Shouldn't this return the error? I don't think we would expect this to happen... right?

Reply:
fixed, an error is not expected